src/test/java/de/bsi/secvisogram/csaf_cms_backend/rest/AdvisoryControllerTest.java

```diff
@@ -168,19 +163,6 @@ void changeCsafDocumentTest_notExisting() throws Exception {
                 .andExpect(status().isNotFound());
     }
 
-    @Test
-    void changeCsafDocumentTest_invalidId() throws Exception {
-
-        String invalidId = "not an UUID";
-
-        this.mockMvc.perform(patch(advisoryRoute + invalidId).with(csrf())
-                        .content(fullAdvisoryJsonString)
-                        .contentType(MediaType.APPLICATION_JSON)
-                        .param("revision", revision))
-                .andDo(print())
-                .andExpect(status().isBadRequest());
-    }
-
```

Comment:
Is this test removed because the update should not be done with a full advisory json? -The corresponding tests for updating advisories should probably not use the `fullAdvisoryJsonString` as input in that case. :thinking:

Reply:
i added a the check for the uuid format again